deploy/charts/venafi-kubernetes-agent/values.yaml

```diff
@@ -57,7 +57,7 @@ image:
   # -- Defaults to only pull if not already present
   pullPolicy: IfNotPresent
   # -- Overrides the image tag whose default is the chart appVersion
-  tag: "v0.1.49"
+  # tag: "v0.0.0"
```

Comment:
Commented out because I wanted to rely on the default which is to use the version of the chart its self.
This should also slightly simplify the release process.